### DISCUSSION TITLE
Поле с автозаполнением в таблице циклов до не работает а после да

### DISCUSSION THREAD

**Discussion by Leo-ui-528** [totumonline/totum-ru-issues-and-discussions#310] [Как это сделать?] [answered]
Здравствуйте подскажите пожалуйста, как лучше сделать? Есть поля, которые автоматически в таблицу циклов подтягивают значения из других таблиц при заполнении определённого поля кодом select(table: 'napravleniya_deyatelnosti'; field: 'kod'; where: 'id' = #f_napravlenie_deyatelnosti), и если они идут последовательно, то все работает, а мне нужно, чтобы к полю номер, которое находится в самом начале подтягивался номер направления деятельности которое выбирается селектом из другой таблицы, аналогичным кодом и нет, как быть? 
![image](https://user-images.githubusercontent.com/75322878/212072174-46f9694b-d8e2-4187-b44f-2d8920b16721.png)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Вам нужен параметр «Отображать в другом месте» или как-то так

**Comment by Leo-ui-528** [comment]
Получилось, спасибо
